runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -286,6 +305,7 @@ private void onConfigChanged(ConfigChanged event)
 
 	private void killedOne()
 	{
+		addCounter(); //Have to add counter before changing values otherwise it throws a warning
```

Comment:
This should probably be on line 322 because I assume it throws a NPE from trying to setText?